### PR TITLE
README simple fixes after code review

### PR DESCRIPTION
**Pull request by Karina-Murawko-Wisniewska-Mobica**
Fixed minor issues in README, indentation in snippets changed (that's why so many changes are visible).

### CODE REVIEW

File: samples/extensions/extended_dynamic_state2/README.md [hunk 1]
```diff
@@ -23,9 +23,9 @@
 
 ## Overview
 
-This sample demostrates hot to use `VK_EXT_extended_dynamic_state2` extension, which eliminates the need to create multiple pipeline in case of specific different parameters.
+This sample demostrates how to use `VK_EXT_extended_dynamic_state2` extension, which eliminates the need to create multiple pipelines in case of specific different parameters.
```
Comment (Szymon-Rajczakowski) [inline samples/extensions/extended_dynamic_state2/README.md]: demostrates - > demonstrates

File: samples/extensions/extended_dynamic_state2/README.md [hunk 2]
```diff
@@ -42,336 +42,336 @@ Previously developers had to create multiple pipelines for different parameters
...
+	VK_DYNAMIC_STATE_RASTERIZER_DISCARD_ENABLE_EXT,
+	VK_DYNAMIC_STATE_PRIMITIVE_RESTART_ENABLE_EXT,
+};
+VkPipelineDynamicStateCreateInfo dynamic_state =
+	vkb::initializers::pipeline_dynamic_state_create_info(
+	    dynamic_state_enables.data(),
+	    static_cast<uint32_t>(dynamic_state_enables.size()),
+	    0);
+
+// Vertex bindings an attributes for model rendering
```
Comment (Szymon-Rajczakowski) [inline samples/extensions/extended_dynamic_state2/README.md]: I think it should be:
// Vertex bindings and attributes for model rendering

File: samples/extensions/extended_dynamic_state2/README.md [hunk 3]
```diff
@@ -42,336 +42,336 @@ Previously developers had to create multiple pipelines for different parameters
...
+In approach above, if we would want to change the patch control points number, then for each different number we would need to create a new pipeline.
 
-However, with dynamic state 2 the number of pipelines can be reduced because of possibility to change parameters of `Depth Bias`, `Primitive Restart`, `Rasterizer Discard` and `Patch Control Points` by calling `cmdSetDepthBiasEnableEXT`, `cmdSetPrimitiveRestartEnableEXT`, `cmdSetRasterizerDiscardEnableEXT` and `cmdSetPatchControlPointsEXT` before `draw_model`.
+However, with 'VK_EXT_extended_dynamic_state2' the number of pipelines can be reduced by the possibility to change parameters of `Depth Bias`, `Primitive Restart`, `Rasterizer Discard` and `Patch Control Points` by calling `vkCmdSetDepthBiasEnableEXT`, `vkCmdSetPrimitiveRestartEnableEXT`, `vkCmdSetRasterizerDiscardEnableEXT` and `vkCmdSetPatchControlPointsEXT` respectively before calling `draw_model`.
 
-With usage of above functions we can reduce number of pipelines.
+With the usage of above functions we can reduce number of pipelines.
```
Comment (Szymon-Rajczakowski) [inline samples/extensions/extended_dynamic_state2/README.md]: I think it should be "the number of pipelines"

File: samples/extensions/extended_dynamic_state2/README.md (hunk 2 above)
Comment (Karina-Murawko-Wisniewska-Mobica) [reply samples/extensions/extended_dynamic_state2/README.md]: fixed, the same change added in .cpp file

File: samples/extensions/extended_dynamic_state2/README.md (hunk 3 above)
Comment (Karina-Murawko-Wisniewska-Mobica) [reply samples/extensions/extended_dynamic_state2/README.md]: fixed

File: samples/extensions/extended_dynamic_state2/README.md (hunk 1 above)
Comment (Karina-Murawko-Wisniewska-Mobica) [reply samples/extensions/extended_dynamic_state2/README.md]: fixed

**Review by Szymon-Rajczakowski — APPROVED** [review]
LGTM

**Pull request merged**